api/provisioner/machine.go

```diff
@@ -532,3 +537,60 @@ func (m *Machine) SetSupportedContainers(containerTypes ...instance.ContainerTyp
 func (m *Machine) SupportsNoContainers() error {
 	return m.SetSupportedContainers([]instance.ContainerType{}...)
 }
+
+type ProfileChangeInfo struct {
+	AppName        string
+	OldProfileName string
+	CharmRev       int
+	Profile        charm.LXDProfile
```

Comment:
I think we need to be more consistent with our names - `LXDProfile`, `CharmProfile`, `Profile`, in this case, I think it should be `LXDProfile`.

Reply:
updated